### PR TITLE
Released marathon 1.4.2

### PR DESCRIPTION
**Pull request by zen-dog**

### CODE REVIEW

File: repo/packages/M/marathon/9/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    {{#marathon.ssl-keystore-path}}
+    {{#marathon.ssl-keystore-password}}
+    , {{#marathon.https-port}}{{marathon.https-port}}{{/marathon.https-port}}{{^marathon.https-port}}0{{/marathon.https-port}}
+    {{/marathon.ssl-keystore-password}}
+    {{/marathon.ssl-keystore-path}}
+  ],
+  "uris": {{service.uris}},
+  "healthChecks": [
+    {
+      "gracePeriodSeconds": 120,
```
Comment (aquamatthias) [inline repo/packages/M/marathon/9/marathon.json.mustache]: should be 1800

File: repo/packages/M/marathon/9/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion" : "3.0",
+  "name" : "marathon",
+  "version" : "1.4.2",
+  "minDcosReleaseVersion" : "1.8",
```
Comment (aquamatthias) [inline repo/packages/M/marathon/9/package.json]: Should be 1.9

**Comment by zen-dog** [comment]
Done

**Comment by joerg84** [comment]
LGTM

**Comment by joerg84** [comment]
@ryadav88 I reviewed and will merge this.

**Pull request merged**